config/sidekiq.yml

```diff
@@ -7,3 +7,4 @@
   - [last_active,  1]
```

Comment:
Add link to documentation https://github.com/mperham/sidekiq/wiki/Advanced-Options#queues